lib/message/encrypt.js

```diff
@@ -30,40 +57,34 @@ export default async function encryptMessage({
     }
 
     if (detached) {
-        // Add cleartext signature for backward compatibility
-        const { message } = sanitizedOptions;
-        if (message.fromStream) {
-            // TODO clone input stream -- sanitizedOptions.message.fromStream ? await readMessage({ clone(<input data>) }) : sanitizedOptions.message;
-            throw new Error('Encrypting and detached-signing a message created from a stream is not supported yet');
+        if (!armor && isStream(textData || binaryData)) {
+            throw new Error('Unsupported detached signature when streaming data and requesting non-armored result');
```

Comment:
```suggestion
        if (!armor && isStream(data)) {
            throw new Error('Creating unarmored detached signatures when streaming data is not supported');
```
Also, why is this not supported, actually? (Not that it's important, just curious)

Reply:
RIght now it's either armored or object output, and with the latter there was some issue with pulling the streams in the resulting signatures IIRC (we had a call about this and opted for not supporting this case for simplicity).

Reply:
OK :+1: